src/python/WMCore/JobSplitting/JobFactory.py

```diff
@@ -420,3 +419,26 @@ def getPerformanceParameters(self, defaultParams):
         sizePerEvent = defaultParams.get('sizePerEvent', None) or 0
         memory = defaultParams.get('memoryRequirement', None) or 0
         return timePerEvent, sizePerEvent, memory
+    
+    def checkCreateFailConditoin(self, failedJob, failedReason):
+
+        valueMap = {"MaxWallTimeHours": "estimatedJobTime", 
+                    "MaxMemory": "estimatedMemoryUsage", 
```

Comment:
Yes, this would result in the same thing.

However, to illustrate the macro expansion,

foo = 1234
request_disk = $(foo)

is valid, while:

foo = 1234
+RequestDisk = $(foo)

is not.

Another difference is the handling of quoting rules.  The following two are *not* equivalent:

error = foo.err
+Err = foo.err

as "foo.err" should be a string and needs quoting:

+Err = "foo.err"

(Historical note: the submit file format predates ClassAds.  The "+" syntax was adopted to give the flexibility of using either format.  Most "normal users" probably find the submit file format preferable.)